### PR TITLE
Efi cleaning

### PR DESCRIPTION
**Pull request by jreidinger**
proposal how fix can look like. Welcome any comments

### CODE REVIEW

File: libstorage/src/Disk.cc [hunk 1]
```diff
@@ -2113,6 +2113,17 @@ int Disk::doRemove( Volume* v )
 	    getStorage()->removeDmMapsTo( getPartDevice(p->OrigNr()) );
 	    ret = v->prepareRemove();
 	    }
+        // before deleting partitions ensure that efi do not contain it
+	if( ret==0 && !p->created() )
+            {
+	    std::ostringstream cmd_line;
+            cmd_line.str("");
```
Comment (aschnell) [inline libstorage/src/Disk.cc]: This is useless, see http://www.cplusplus.com/reference/sstream/ostringstream/ostringstream/.

Comment (jreidinger) [inline libstorage/src/Disk.cc]: you are right, it is leftover of clearing cmd_line from previous attempt

**Comment by aschnell** [comment]
Changelog is missing.

**Comment by jreidinger** [comment]
ok, I will add it, I firstly like to discuss approach

**Comment by shundhammer** [comment]
The change log looks good for me. As for the functional part of the patch, Arvin already commented on it.

**Comment by jreidinger** [comment]
thanks both of you merging.

**Pull request merged**